src/api/spec/controllers/person/token_controller_spec.rb

```diff
@@ -0,0 +1,105 @@
+require 'rails_helper'
+# WARNING: If you change tests make sure you uncomment this line
+# and start a test backend. Some of the actions
+# require real backend answers for projects/packages.
+# CONFIG['global_write_through'] = true
```

Comment:
Why do you have those comments if there are no backend calls that provide meaningful response (a.k.a. `vcr: false`)?

Reply:
Right, I shall drop them. I just c&p'ed this from the preson controller spec